### ISSUE TITLE
can't reproduce your results

### ISSUE THREAD

**Issue by wenhuchen** [uoguelph-mlrg/confidence_estimation#1]
Hi, I'm running your code with the following command:
```
python train.py --dataset cifar10 --model vgg13 --budget 0.3 --data_augmentation --cutout 16
```
But the accuracy on CIFAR10 only achives 78%, while the normal accuracy should be around 95%. Is there anything that I missed?

**Comment by TDeVries** [comment]
That's strange, I haven't had any problems like this before. What version of pytorch are you using?

**Comment by wenhuchen** [comment]
Hi, it's my bad, I assumed it converged within 30 minutes. Now, after I trained for 3 hours for 180 epochs, I am able to reproduce your reported results.

**Comment by TDeVries** [comment]
Great! Thanks for reporting back.
